src/components/CreateUserForm.vue

```
@@ -0,0 +1,144 @@
+const birthDateFormatted = ref(formatDate(new Date(firstValidDoB.value)));
+
+const roles = [
+  { title: 'HAK', value: UserRole.HAK },
+  { title: 'Admin', value: UserRole.Admin },
+  { title: 'Osoba', value: UserRole.Osoba },
+  { title: 'Firma', value: UserRole.Firma },
+  { title: 'Policija', value: UserRole.Policija },
+  { title: 'Super Admin', value: UserRole.SuperAdmin }
+];
+
+function updateUser(field: keyof User, value: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)